src/react-hooks.js

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react';
```

Comment:
Suggestion: what do you think about naming this file with something less generic? It almost look like a library. I don't know, maybe `custom-hooks` or `terraso-hooks`?

Reply:
@cacarrara Yes, I struggled a little with that name, I don't like `react-hooks` is too generic, I agree. I think I like `custom-hooks` I think that one is at least more explicit that it is a bucket for custom hooks, and maybe if eventually there is a way to group the hooks better that file can be split